ods_ci/tasks/Resources/Provisioning/GPU/amd_operator/kmm_operator_install.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-kmm
+
+---
+
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-kmm-operator-group
+  namespace: openshift-kmm
```

Comment:
isn't "openshift" a protected name for namespaces? meaning we cannot create namespaces starting with "openshift"? maybe I misremember

Reply:
I don't think so

Reply:
checked, the limitation is only on OpenShift Projects, not namespaces